tests/input/inputfiletest.py

```diff
@@ -0,0 +1,26 @@
+import pypact as pp
+
+
+def test_reading_in_mass():
```

Comment:
This is great, but to keep things a little consistent, it might help to use `unittest` lib here, or use the silly base class, like is done for the other tests i.e.: https://github.com/fispact/pypact/blob/b29640f6aebd5ad83e19c58997c284eb48e70aa5/tests/input/inputdatatest.py.

For it to then be picked up we would need it to be added to the suite here too: https://github.com/fispact/pypact/blob/b29640f6aebd5ad83e19c58997c284eb48e70aa5/tests/testsuite.py#L19